andela/titan-dev-ah-frontend
# #163518643 Reset Password

#### What does this PR do?
Add password reset feature
#### Description of Task to be completed?
- Convert HTML mockup into react components
- Create Reset Password action creators and reducers
- Connect redux store with reset password component
- Add tests for reset password component, action creators, and reducers
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-reset-password-163518643 `
- clone the backend repo and check out the update branch `ch-server-message-165307938`
- Add the following env variables ```FRONT_END_SERVER_HOST = "http://localhost:8000"```
- run `yarn start` and navigate to /reset_password in your browser
- Typing a real registered email to get reset password link
- Check your email for reset password link
#### Any background context you want to provide?
You have to have an existing account with real email, and your backend server running
#### What are the relevant pivotal tracker stories?
#163518643
